node/src/neighborhood/mod.rs

```diff
@@ -435,23 +433,23 @@ enum RouteDirection {
 impl Neighborhood {
     pub fn new(cryptde: &'static dyn CryptDE, config: &BootstrapperConfig) -> Self {
         let neighborhood_config = &config.neighborhood_config;
-        if neighborhood_config.mode.is_zero_hop()
-            && !neighborhood_config.mode.neighbor_configs().is_empty()
-        {
+        let min_hops_count = neighborhood_config.min_hops_count;
+        let neighborhood_mode = &neighborhood_config.mode;
+        let mode: NeighborhoodModeLight = neighborhood_mode.into();
+        let neighborhood_configs = neighborhood_mode.neighbor_configs();
```

Comment:
Maybe `neighbor_configs` instead of `neighborhood_configs`

Reply:
I've created this card - https://github.com/MASQ-Project/MASQ-Node-issues/issues/704

Although, I didn't mentioned what should be a more accurate routing behaviour. I need your help in postulating some ideas for that.